docs/framework/sitemap.txt

```diff
@@ -1,4 +1,6 @@
 index.md
+	concepts/index.md
```

Comment:
So I can understand how to organize this, what else do you envision being part of the "Concepts" section?